docs/ConverterUsage.md

```diff
@@ -0,0 +1,119 @@
+# Converter usage & demo
```

Comment:
Synopsis is missing: what is the converter, what it does, where to find it.

Reply:
Addressed in [adc296d](https://github.com/async-profiler/async-profiler/pull/1029/commits/adc296d22ffe7bcf227f89b9b92e799a2f72f2f0)